app/models/grda_warehouse_base.rb

```diff
@@ -55,4 +55,16 @@ def paper_trail_meta_value(key)
       value
     end
   end
+
+  def self.references_hud_client?
+    columns_hash.key?('PersonalID') && reflect_on_association(:client)
+  end
+
+  def self.references_hud_enrollment?
+    columns_hash.key?('EnrollmentID') && reflect_on_association(:group)
```

Comment:
Should this be `:group` or `:enrollment`?  I'm not familiar with the `:group` relation.

Reply:
I think this should be `:enrollment`

Reply:
Yes, should be `:enrollment`, sorry I missed this typo. Thanks for catching!